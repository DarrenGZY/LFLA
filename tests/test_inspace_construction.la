function main()
{
     vector v1 = [1,0,0];
    vector v2 = [0,1,0];
    vector v3 = [0,0,1];

    matrix mat = [1,0,0;0,1,0;0,0,1];
    vector vecs[3] = {v1, v2, v3};
    inspace ins = inspace(vecs, mat);
    vector v4 = [2,3,4];
    vector v5 = [-1,1,2];
    print( ins<<v4,v5>>);	
}
