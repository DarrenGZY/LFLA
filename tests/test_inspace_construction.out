Please check dimensions of vectors and matrix
